Update array-jobs.md (#78)
ImperialCollegeLondon/icl-rcs-user-guide: docs/hpc/queues/job-sizing-guidance.md

| [gpu72](#gpu72) | Main queue for gpu jobs* | 1 | 1 - 64 | 1 - 920 | 0 - 72 |
| [capability24](#capability) | Multi-node jobs 24h | 2 - 4 | 1 - 64 | 1 - 450 | 0 - 24 |
| [capability48](#capability) | Multi-node jobs 48h | 2 - 4 | 1 - 64 | 1 - 450 | 24 - 48 |
| [jupyter](#jupyter) | Queue for JupyterHub jobs* | 1 | 1, 4, 8 | 8, 32, 64 | 2, 4, 8 |
| [jupytergpu](#jupytergpu) | Queue for JupyterHub GPU jobs* | 1 | 4 | 32 | 8 |
| [ood](#ood) | Queue for Open OnDemand* | 1 | 1, 4, 8 | 8, 32, 64 | 2, 4, 8 |

\* Please see details for specific queues below as there may be additional restrictions or limitations.

#### small
This queue is for low-core jobs, up to 16 cores and 128gb of RAM. Jobs in this queue typically make use of OpenMP or multi-threading to parallelise workflows.

#### medium

> This queue is jobs using up-to 64 cores and/or 450GB of RAM. This is half an AMD EPIC 7742 node or all of an Intel IceLake.

Reviewer: ```suggestion
This queue is jobs using up-to 64 cores and/or 450GB of RAM. This is half an AMD EPYC 7742 node or all of an Intel Ice Lake.
```

Author: Correct spelling is EPYC 
https://www.amd.com/en/products/processors/server/epyc.html

```suggestion
This queue is jobs using up-to 64 cores and/or 450GB of RAM. This is half an AMD EPYC 7742 node or all of an Intel Ice Lake node.
```

#### capability
This queue is for larger multi-node jobs, using up to 4 entire nodes at once. Workflows in this queue will normally be using technologies such as MPI to communicate between all nodes.

#### jupyter

> This queue is where [JupyterHub](https://jupyterhub-11.rcs.ic.ac.uk) jobs are run. There is a limit of 1 concurrent job per user across both jupyter queues.

Reviewer: ```suggestion
This queue is where [JupyterHub](https://jupyter.cx3.rcs.ic.ac.uk/) jobs are run. There is a limit of 1 concurrent job per user across both jupyter queues.
```

#### jupytergpu
This queue has NVIDIA A40 (48GB) and NVIDIA RTX6000 (24GB) GPU's. There is a limit of 1 concurrent job per user across both jupyter queues.

#### ood

> This is the queue for [Open OnDemand](../applications/guides/openondemand/). This service allows users to run R-studio on our compute nodes via a web portal. Only 1 job can be run at a time.

Reviewer: ```suggestion
This is the queue for [Open OnDemand](../../applications/guides/openondemand/). This service allows users to run R-studio on our compute nodes via a web portal. Only 1 job can be run at a time.
```

Author: ```suggestion
This is the queue for [Open OnDemand](../../applications/guides/openondemand/). This service allows users to run R-studio on our compute nodes via the [web portal](https://openondemand.rcs.ic.ac.uk). Only 1 job can be run at a time.
```

```
You should not request an interactive job longer than 8 hours, and should make sure to end your session once you are done as to not leave it idle.

**Note that some CPU nodes have up to 128 cores and 920GB of RAM (cpu_type=rome), while some only have up to 64 cores and 450GB of RAM (cpu_type=icelake). This means both large and medium queues could be considered whole node queues depending on what CPU type is requested.

## CX3 cluster Limits

> For ensure fair access to resources for all researchers there are a couple of limits that apply across all queues:

Reviewer: ```suggestion
For ensure fair access to resources for all researchers there are a couple of limits that apply across all queues:

```

Leave a space for correct rendering of MD.

Author: ```suggestion
To ensure fair access to resources for all researchers there are a couple of limits that apply across all queues:

```
